app/src/organisms/RunPreview/index.tsx

```diff
@@ -81,10 +80,24 @@ export const RunPreviewComponent = (
     isCurrentCommandVisible,
     setIsCurrentCommandVisible,
   ] = React.useState<boolean>(true)
-  if (robotSideAnalysis == null) return null
+  const filteredCommandsFromQuery = React.useMemo(
+    () =>
+      commandsFromQuery?.filter(
+        command =>
+          !('intent' in command && command.intent === 'fixit') ||
+          !('intent' in command)
```

Comment:
Nit, after thinking about this some more: I think this might be a bit easier. I'd test it just to make sure:
```suggestion
!('intent' in command) || command.intent !== 'fixit'
```

Reply:
tested this! works as expected